crates/runtime/src/accelerated_table.rs

```diff
@@ -273,6 +279,31 @@ impl Builder {
         self
     }
 
+    /// Set the existing full refresh mode accelerated table to synchronize with after the initial load completes
+    ///
+    /// A full table scan of the existing accelerated table is required to initialize a synchronized accelerated table.
+    ///
+    /// Handling append/changes mode should be possible, but requires more care to ensure
+    /// that delta updates are applied correctly after the initial table scan.
+    pub async fn synchronize_with(
+        &mut self,
+        existing_accelerated_table: &AcceleratedTable,
+    ) -> AcceleratedTableBuilderResult<&mut Self> {
+        ensure!(
```

Comment:
The reason to restrict this to `RefreshMode::Full` is that we would lose any updates from the time the initial load of the locapod accelerator starts, until when it finishes. That could leave the datasets in an inconsistent state.

It should be possible to handle this, but will require the parent dataset to know when to start buffering updates and the synchronize to trigger a flush of all pending updates.